### ISSUE TITLE
[Pre-release] Create a standalone content validation script

### ISSUE THREAD

**Issue by cvalarida** [department-of-veterans-affairs/va.gov-team#7953]
## Problem Statement
Currently, we run content validation in the application CI pipeline, but not in the content-only deploy pipeline. As we separate the responsibilities of the content and application builds and deploys, we need to continue running content validation on a regular schedule.

This is step one: Create a standalone script that isn't run during any build.

## Background
Ideally, the content validation script would be run during the content build and cancel the deploy if content is found invalid. However, because there will be hundreds of content editors, it's infeasible to invalidate the deployment of _all_ content because one editor introduced a violation. It's important to note **this is the same as it is today.** The only difference is that this validation will no longer be happening during the application build and deploy. It still needs to be done, so the solution is to do this in a separate, scheduled job.

## Goal
A standalone script validates the contents of a build directory with pre-built files (HTML, JS, etc.).

---
## Tasks
- [ ] Determine all the content validation we currently do
    - Broken links
    - Accessibility checks
    - For any other content validation found, update https://github.com/department-of-veterans-affairs/va.gov-team/issues/7966
      - I think these two are it, but just so our bases are covered... :man_shrugging: 
- [ ] Write a standalone script which performs all the content validation
    - This can live in `vets-website` for now, though we may move it later

## Acceptance Criteria
- [ ] Current content validation is unchanged
    - That'll come later
- [ ] A standalone script is created which validates all content in a directory passed as an argument to it

**Comment by cvalarida** [comment]
**Note:** Depending on the complexity of the migration, it might make sense to break this down into more tickets.

**Comment by billfienberg** [comment]
Content Validation Design Doc: https://github.com/department-of-veterans-affairs/va.gov-team/pull/9658

**Comment by timwright12** [comment]
Noting that the broken link checker should still be part of the normal build. as outlined in the design doc (https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/7953)

**Comment by timwright12** [comment]
**Working branch**: feature/7953-create-content-validation-script
**PR**: https://github.com/department-of-veterans-affairs/content-build/pull/39

**Comment by Clark-W-Griswold** [comment]
Tim:
A11y tests passing in Jenkins
e2e failing because no CSS/JS is being applied in the Docker container
Next: get e2e passing, then isolate the job to be run on demand

**Comment by timwright12** [comment]
Update:
All tests passing in Jenkins, merged into master
A placeholder in Circle has also been added to run a test hourly

To Do:
Move tests from Jenkins in to CircleCI
 - Link checker should work fine
 - nightwatch tests will need some work, but should be installed in the placeholder job that's set to run hourly on master (hourly-content-validation)

**Comment by Clark-W-Griswold** [comment]
Note from Megan — Dan and I brought this up to Dror and I’m going to discuss with Fleet today. If we’re seeking to change this behavior (make it not block deploy) we won’t get final answer/approval on that until mid-January, is my guess. So if that is blocking to us we may need to make a call on moving forward with current behavior for now. I’ll let you know if anything different comes up in my convo w/ Fleet.

**Comment by timwright12** [comment]
We finished up the script work, waiting on Ops to activate the schedule job in Jenkins. We also broke off Circle into another PR to complete later https://github.com/department-of-veterans-affairs/content-build/pull/43

Ops is lending a casual hand with getting the scheduled job into Circle, so it that works we'll leave it there, but the current plan is still to get it into Jenkins

Slack thread for context https://dsva.slack.com/archives/CBU0KDSB1/p1609356529385600

**Comment by timwright12** [comment]
Noticing this is complete because we have the schedule script in another ticket https://app.zenhub.com/workspaces/vsp---frontend-tools-5fc9325744944e0015ed1861/issues/department-of-veterans-affairs/va.gov-team/7964

**Comment by meganhkelley** [comment]
Standalone validation includes end-to-end accessibility tests and 508 tests — broken link checking is left as part of the build process.

**Comment by timwright12** [comment]
This is complete, we have a script running in Jenkins and Circle currently

Jenkins: http://jenkins.vfs.va.gov/job/testing/job/content-build-integration-testing/
Circle: https://app.circleci.com/pipelines/github/department-of-veterans-affairs/content-build/815/workflows/851b1150-b43d-49ca-9514-27e607e6d552
